grafana/grafana
# Annotations: Fix rendering

Before

https://user-images.githubusercontent.com/88068998/236911654-b4acfba8-2876-4109-993e-ed734fce7daf.mov


After

https://user-images.githubusercontent.com/88068998/236911691-25f56fbd-4f2f-4c4a-aa7a-aaff85a0f53f.mov


Fixes #66625 

**Special notes for your reviewer:**
Applies to Time Series, State Timeline and Candlestick.


**Testing steps:**

1. Create a dashboard and add visualizations: Time series, Candlestick, State timeline
2. Create 1-2 annotations 
2a. Check enabling/disabling annotations
2b. Check deleting an annotation
3. The annotations should correctly appear in panels (also in preview)

Please check that:
